include/openthread/platform/radio.h

```diff
@@ -341,9 +349,10 @@ typedef struct otRadioFrame
              * it must also set this flag before passing the frame back from the `otPlatRadioTxDone()` callback.
              */
             bool mIsHeaderUpdated : 1;
-            bool mIsARetx : 1;             ///< Indicates whether the frame is a retransmission or not.
-            bool mCsmaCaEnabled : 1;       ///< Set to true to enable CSMA-CA for this packet, false otherwise.
-            bool mCslPresent : 1;          ///< Set to true if CSL header IE is present.
+            bool mIsARetx : 1;       ///< Indicates whether the frame is a retransmission or not.
+            bool mCsmaCaEnabled : 1; ///< Set to true to enable CSMA-CA for this packet, false to disable both CSMA
+                                     ///< backoff and CCA hence ignoring mMaxCsmaBackoffs.
```

Comment:
Since the doc spans multi-line we can use `/** */` format and explain more:
```c++
            /**
             * Set to `true` to enable CSMA-CA for this frame, `false` to disable both CSMA backoff and CCA.  
             *
             * When it is set to `false`, the frame MUST be sent without performing CCA. In this case 
             * `mMaxCsmaBackoffs` value MUST also be ignored.
             */
            bool mCsmaCaEnabled : 1; 
```